<ui version="4.0" >
 <class>WengoPhoneWindow</class>
 <widget class="QMainWindow" name="WengoPhoneWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WengoPhone</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="profileBar" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex" >
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QTabWidget" name="tabWidget" >
         <property name="autoFillBackground" >
          <bool>false</bool>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabContactList" >
          <attribute name="title" >
           <string>Contacts</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabHistory" >
          <attribute name="title" >
           <string>History</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabDialpad" >
          <attribute name="title" >
           <string>Dialpad</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="callBarFrame" >
      <property name="minimumSize" >
       <size>
        <width>16</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>65</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="configPanelFrame" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>417</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuContacts" >
    <property name="title" >
     <string>&amp;Contacts</string>
    </property>
    <addaction name="actionAddContact" />
    <addaction name="actionSearchWengoContact" />
    <addaction name="separator" />
    <addaction name="actionShowHideOfflineContacts" />
    <addaction name="actionShowHideContactGroups" />
   </widget>
   <widget class="QMenu" name="menuWengo" >
    <property name="title" >
     <string>&amp;Wengo</string>
    </property>
    <addaction name="actionEditMyProfile" />
    <addaction name="actionLogOff" />
    <addaction name="separator" />
    <addaction name="actionShowWengoAccount" />
    <addaction name="actionWengoServices" />
    <addaction name="separator" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuActions" >
    <property name="title" >
     <string>&amp;Actions</string>
    </property>
    <addaction name="actionCreateConferenceCall" />
    <addaction name="separator" />
    <addaction name="actionSendSms" />
    <addaction name="separator" />
    <addaction name="actionAcceptCall" />
    <addaction name="actionHangUpCall" />
    <addaction name="actionHoldResumeCall" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionShowWengoForum" />
    <addaction name="actionWengoFAQ" />
    <addaction name="separator" />
    <addaction name="actionShowAbout" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuClearHistory" >
     <property name="title" >
      <string>Clear &amp;History</string>
     </property>
     <property name="icon" >
      <iconset/>
     </property>
     <addaction name="actionClearOutgoingCalls" />
     <addaction name="actionClearIncomingCalls" />
     <addaction name="actionClearMissedCalls" />
     <addaction name="actionClearRejectedCalls" />
     <addaction name="actionClearChatSessions" />
     <addaction name="actionClearSMS" />
     <addaction name="separator" />
     <addaction name="actionClearAll" />
    </widget>
    <addaction name="actionOpenChatWindow" />
    <addaction name="actionShowFileTransfer" />
    <addaction name="actionHideMainWindowToolBar" />
    <addaction name="separator" />
    <addaction name="actionToggleVideo" />
    <addaction name="actionShowConfig" />
    <addaction name="separator" />
    <addaction name="menuClearHistory" />
   </widget>
   <addaction name="menuWengo" />
   <addaction name="menuContacts" />
   <addaction name="menuActions" />
   <addaction name="menuTools" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="contextMenuPolicy" >
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="windowTitle" >
    <string>ToolBar</string>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionAddContact" />
   <addaction name="actionSearchWengoContact" />
   <addaction name="actionCreateConferenceCall" />
   <addaction name="actionSendSms" />
  </widget>
  <action name="actionShowWengoAccount" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>View My &amp;Wengo Account...</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionShowWengoForum" >
   <property name="text" >
    <string>Wengo &amp;Forum...</string>
   </property>
  </action>
  <action name="actionShowAbout" >
   <property name="text" >
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionAddContact" >
   <property name="icon" >
    <iconset resource="wengophone.qrc" >:/pics/iconbar/add_contact.png</iconset>
   </property>
   <property name="text" >
    <string>Add &amp;Contact...</string>
   </property>
   <property name="iconText" >
    <string>Add Contact</string>
   </property>
   <property name="toolTip" >
    <string>Add Contact</string>
   </property>
  </action>
  <action name="actionShowConfig" >
   <property name="text" >
    <string>&amp;Configuration...</string>
   </property>
  </action>
  <action name="actionSendSms" >
   <property name="icon" >
    <iconset resource="wengophone.qrc" >:/pics/iconbar/sms.png</iconset>
   </property>
   <property name="text" >
    <string>Send SMS...</string>
   </property>
   <property name="iconText" >
    <string>Send SMS</string>
   </property>
   <property name="toolTip" >
    <string>Send SMS</string>
   </property>
  </action>
  <action name="actionEditMyProfile" >
   <property name="icon" >
    <iconset resource="wengophone.qrc" >:/pics/config/imaccounts.png</iconset>
   </property>
   <property name="text" >
    <string>View My &amp;Profile...</string>
   </property>
   <property name="iconText" >
    <string>View My Profile</string>
   </property>
   <property name="toolTip" >
    <string>View My Profile</string>
   </property>
  </action>
  <action name="actionCreateConferenceCall" >
   <property name="icon" >
    <iconset resource="wengophone.qrc" >:/pics/iconbar/conference.png</iconset>
   </property>
   <property name="text" >
    <string>Start &amp;Conference...</string>
   </property>
   <property name="iconText" >
    <string>Start Conference</string>
   </property>
   <property name="toolTip" >
    <string>Start Conference</string>
   </property>
  </action>
  <action name="actionWengoFAQ" >
   <property name="text" >
    <string>Wengo &amp;Wiki/FAQ...</string>
   </property>
  </action>
  <action name="actionWengoServices" >
   <property name="text" >
    <string>Learn more about &amp;Wengo Services...</string>
   </property>
  </action>
  <action name="actionShowHideOfflineContacts" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Show/Hide Unavailable &amp;Contacts</string>
   </property>
  </action>
  <action name="actionClearChatSessions" >
   <property name="text" >
    <string>&amp;Chat Sessions</string>
   </property>
  </action>
  <action name="actionClearAll" >
   <property name="text" >
    <string>&amp;All</string>
   </property>
  </action>
  <action name="actionSearchWengoContact" >
   <property name="icon" >
    <iconset resource="wengophone.qrc" >:/pics/iconbar/search_contact.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Find Contact...</string>
   </property>
   <property name="iconText" >
    <string>Find Contact</string>
   </property>
   <property name="toolTip" >
    <string>Find Wengo Contacts</string>
   </property>
  </action>
  <action name="actionAcceptCall" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>&amp;Accept</string>
   </property>
  </action>
  <action name="actionHangUpCall" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>&amp;HangUp</string>
   </property>
  </action>
  <action name="actionHoldResumeCall" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>&amp;Hold/Resume</string>
   </property>
  </action>
  <action name="actionShowHideContactGroups" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Show/Hide &amp;Groups</string>
   </property>
  </action>
  <action name="actionLogOff" >
   <property name="text" >
    <string>Log &amp;Off...</string>
   </property>
   <property name="iconText" >
    <string>Log Off</string>
   </property>
   <property name="toolTip" >
    <string>Log Off</string>
   </property>
  </action>
  <action name="actionOpenChatWindow" >
   <property name="icon" >
    <iconset resource="wengophone.qrc" >:/pics/iconbar/chat.png</iconset>
   </property>
   <property name="text" >
    <string>Show &amp;Chat Window</string>
   </property>
   <property name="iconText" >
    <string>Show Chat Window</string>
   </property>
   <property name="toolTip" >
    <string>Show Chat Window</string>
   </property>
  </action>
  <action name="actionClearOutgoingCalls" >
   <property name="text" >
    <string>&amp;Outgoing Calls</string>
   </property>
  </action>
  <action name="actionClearIncomingCalls" >
   <property name="text" >
    <string>&amp;Incoming Calls</string>
   </property>
  </action>
  <action name="actionClearMissedCalls" >
   <property name="text" >
    <string>&amp;Missed Calls</string>
   </property>
  </action>
  <action name="actionToggleVideo" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Enable/Disable &amp;Video Calls</string>
   </property>
  </action>
  <action name="actionShowVolumePanel" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="wengophone.qrc" >:/pics/statusbar/audio_status_ok.png</iconset>
   </property>
   <property name="text" >
    <string>Show &amp;Volume Panel</string>
   </property>
  </action>
  <action name="actionClearSMS" >
   <property name="text" >
    <string>Short Text &amp;Messages (SMS)</string>
   </property>
  </action>
  <action name="actionShowFileTransfer" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Show &amp;File Transfer</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="actionClearRejectedCalls" >
   <property name="text" >
    <string>Rejected Calls</string>
   </property>
  </action>
  <action name="actionHideMainWindowToolBar" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Hide &amp;ToolBar</string>
   </property>
  </action>
  <action name="actionClear_chat_history" >
   <property name="text" >
    <string>clear chat history</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="wengophone.qrc" />
 </resources>
 <connections/>
</ui>
